packages/realm/src/Collection.ts

```diff
@@ -39,6 +41,20 @@ export abstract class Collection<T = unknown, ChangeCallbackType = unknown> impl
     throw new Error("Not yet implemented");
   }
 
+  toJSON(_?: string, cache = new Map()): Array<DefaultObject> {
+    // return this.map((item, index) =>
```

Comment:
original implementation used map, since we do not have it defined I just went with regular loop

Reply:
I'd probably want to more this to `OrderedCollection` and then you'd be able to use the `map` method.